src/Module.php

```diff
@@ -40,6 +40,11 @@ public function getViewHelperConfig()
                     /** @var Rollout $rollout */
                     $rollout = $serviceLocator->get('zf2_rollout');
                     return new View\Helper\IsActive($rollout);
+                },
+                'rollout_description' => function (AbstractPluginManager $pluginManager) {
+                    $serviceLocator = $pluginManager->getServiceLocator();
+                    $rollout = $serviceLocator->get('zf2_rollout_config');
+                    return new View\Helper\Description($rollout);
```

Comment:
That doesn't seem valid ... it will inject all configurations, while it should only inject the features ... Seeing the code of the Description helper this cannot work as it assumes to receive what is inside the 'features' key

Reply:
Also adapt the name of the variable

Reply:
Good catch !